options.c

```diff
@@ -3728,8 +3728,10 @@ struct fio_option fio_options[FIO_MAX_OPTS] = {
 	{
 		.name	= "max_latency",
 		.lname	= "Max Latency (usec)",
-		.type	= FIO_OPT_STR_VAL_TIME,
-		.off1	= offsetof(struct thread_options, max_latency),
+		.type	= FIO_OPT_ULL,
```

Comment:
This change was needed to apply limits to trim and it works because is_time is set so the time strings get decoded correctly.